package.json

```diff
@@ -52,7 +52,7 @@
   "scripts": {
     "test": "npm run test:javascript && npm run test:typescript",
     "test:javascript": "jasmine spec/**/**.spec.js",
-    "test:typescript": "tsc examples/typescript/example.ts --noEmit --strict",
+    "test:typescript": "tsc examples/typescript/example.ts --noEmit --strict --skipLibCheck",
```

Comment:
I generally wouldn’t add `--skipLibCheck`, since it can mask real problems in third-party `.d.ts` files (see https://www.typescriptlang.org/tsconfig/#skipLibCheck)

Since this repo is monkey-patched, if the axios upgrade genuinely removes the need for downstream overrides, I’m okay making an exception here.